python/search_completer/search_completer.py

```diff
@@ -313,51 +215,18 @@ def _clear_model(self, add_loading_item=True, add_more_text_item=False):
         self.model().clear()
 
         if add_loading_item:
-            item = QtGui.QStandardItem("Loading data...")
+            item = QtGui.QStandardItem("Hold on, loading search results...")
```

Comment:
I find **Hold on** a bit aggressive, implying that I'm being impatient. Maybe I'm just sensitive. Perhaps just **Loading search results...**?